db/c.cc

```diff
@@ -294,12 +294,13 @@ struct rocksdb_comparator_t : public Comparator {
   void (*destructor_)(void*);
   int (*compare_)(void*, const char* a, size_t alen, const char* b,
                   size_t blen);
-  const char* (*name_)(void*);
   int (*compare_ts_)(void*, const char* a_ts, size_t a_tslen, const char* b_ts,
                      size_t b_tslen);
   int (*compare_without_ts_)(void*, const char* a, size_t alen,
                              unsigned char a_has_ts, const char* b, size_t blen,
                              unsigned char b_has_ts);
+  const char* name_;
+  unsigned char ckwdbcbe_;
```

Comment:
Can we replace this with the full name?

Reply:
The full name on snake case is really really long. It's easy to see what it means with code navigation